hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionName.java

```diff
@@ -20,4 +22,15 @@ public String getName() {
     public TopicName getTopicName() {
         return topicName;
     }
+
+    public static SubscriptionName fromString(String string) {
+        String[] tokens = string.split("\\$");
+        checkArgument(tokens.length > 1, "Incorrect string format. Expected 'topic$subscription'. Found:'%s'", string);
+        return new SubscriptionName(tokens[1], TopicName.fromQualifiedName(tokens[0]));
+    }
+
+    @Override
+    public String toString() {
+        return topicName.qualifiedName() + "$" + name;
```

Comment:
maybe we should stick to the good 'ol consumer group id `{group}_{topic}_{subscription}`, as in kafka's `/consumers` zookeeper path, instead of reinventing a new abstraction :)